.gitignore

```diff
@@ -9,5 +9,6 @@ dist
 .vercel
 .DS_Store
 
+env.d.ts
```

Comment:
this is my fix for the aforementioned gitignore issue.
please double check this before merging.